public/js/src/module/store.js

```diff
@@ -945,5 +963,6 @@ export default {
     dynamicData: dataStore,
     record: recordStore,
     flush,
-    dump
+    dump,
+    get lastSavedRecords() { return server.lastSavedRecords; },
```

Comment:
I tried to keep the changes to `store.js` to a minimum, keeping as much last-saved logic in `last-saved.js` as possible. These minor changes were necessary because the store's schema all has to be defined at initialization time. This breaks a bit with the pattern for other stores, as `last-saved.js` references the `db.js` interfaces directly rather than a wrapper around them.